src/android/CameraPreview.java

```diff
@@ -23,25 +23,26 @@
 
 public class CameraPreview extends CordovaPlugin implements CameraActivity.CameraPreviewListener {
 
-  private final String TAG = "CameraPreview";
-  private final String setColorEffectAction = "setColorEffect";
-  private final String setZoomAction = "setZoom";
-  private final String setFlashModeAction = "setFlashMode";
-  private final String startCameraAction = "startCamera";
-  private final String stopCameraAction = "stopCamera";
-  private final String previewSizeAction = "setPreviewSize";
-  private final String switchCameraAction = "switchCamera";
-  private final String takePictureAction = "takePicture";
-  private final String showCameraAction = "showCamera";
-  private final String hideCameraAction = "hideCamera";
-  private final String getSupportedPictureSizesAction = "getSupportedPictureSizes";
-
-  private final String [] permissions = {
+  private static final String TAG = "CameraPreview";
+
+  private static final String COLOR_EFFECT_ACTION = "setColorEffect";
```

Comment:
This is a pattern I have seen in most cordova plugins.
This places all the functions that are called from javascript in the top which makes it more robust and readable I think.
I would keep it.